docs/ADR/ADR003-github-registry-for-container-image-copied-to-ECR.md

```diff
@@ -0,0 +1,29 @@
+# ADR003: Github registry will be used as container image registry with copies to AWS ECR for each env
+
+Date: 2024-01-29
+
+## Status
+
+  * Accepted
+
+## Context
+
+  * A single image must be deployable to all environments
+  * deployment will require an image registry accessible by app
+  * docker scout should be used in build process and block push based on high/critical issues
+
+## Decision
+
+  * [Github image registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) as central registry
+  * ECR in each AWS env with copy of image for further scanning and integration with AppRunner
+  
+## Considered Options
```

Comment:
Let's mention the consideration using apprunner native runtime -- we'll proceed with docker route for now, but apprunner may reduce moving parts so reduce maintenance long term